notifications/tests/tests.py

```diff
@@ -457,6 +458,31 @@ def test_unread_list_api_mark_as_read(self):  # pylint: disable=invalid-name
                          self.message_count - 2*num_requested)
         self.assertEqual(len(data['unread_list']), num_requested)
 
+    def test_unread_all_objects(self):
+        """
+        Test notification with all objects (actor, target, action_object)
+        Test that object URLs are in the output.
+        """
+        self.login('to', 'pwd')
+        Notification.objects.filter(recipient=self.to_user).mark_all_as_read()
+        to_customer = Customer.objects.create(name="to_customer")
+        action_customer = Customer.objects.create(name="action_customer")
+        from_customer = Customer.objects.create(name="from_customer")
+        target_object = TargetObject.objects.create(name="target_object")
+        notify.send(from_customer, recipient=self.to_user, verb='commented', action_object=action_customer,
+                    target=target_object)
+        response = self.client.get(reverse('notifications:live_unread_notification_list'))
+        data = json.loads(response.content.decode('utf-8'))
+        self.assertEqual(data['unread_count'], 1)
+        notification = data['unread_list'][0]
+        self.assertEqual(notification['actor'], "from_customer")
+        self.assertEqual(notification['action_object_url'], f'foo/{action_customer.id}/')
+        self.assertEqual(notification['target_url'], f'bar/{target_object.id}/')
+        self.assertEqual(notification['actor_url'], f'foo/{from_customer.id}/')
+
+
+
+
```

Comment:
I would remove these blank lines and leave only 1 blank line

Reply:
fixed